setup.py

```diff
@@ -49,11 +49,14 @@
             "tox>=3.5.3,<4.0",
             # linting
             "flake8>=3.0,<4.0",
-            "isort>=4.3,<5.0",
+            "isort>=5.1.4,<6.0",
```

Comment:
I think this is going to kill 2.7

Reply:
I'm being punished for not running `tox` before opening this PR. Yeah, I'll add the version specifier here too... 😢